src/main.rs

```
@@ -10,3 +12,27 @@ fn main() -> Result<()> {
 
     Ok(())
 }
+
+pub struct PreBuild {}
+impl PreBuild {
+    /// When kaps will have its own release on GitHub, the binary could be fetched here.
+    /// Meanwhile, we are using git2 library to clone Kaps from GitHub, then cargo to build it from the sources.
+    pub fn build_kaps(kaps_repository_url: &str) {
```

This should return a proper error, through a `Result`.